### ISSUE TITLE
The README section 'Usage' mentions example code that does not exist.

### ISSUE THREAD

**Issue by HenryHall** [faker-js/faker#2392]
### Pre-Checks

- [X] Follow our [Code of Conduct](https://github.com/faker-js/faker/blob/next/CODE_OF_CONDUCT.md).
- [X] Read the [Contributing Guidelines](https://github.com/faker-js/faker/blob/next/CONTRIBUTING.md).
- [X] Read the [docs](https://fakerjs.dev/guide).
- [X] Check that there isn't [already an issue](https://github.com/faker-js/faker/issues) that reports the same bug to avoid creating a duplicate.
- [X] Make sure this is a Faker issue and not related to a combination with another package.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/faker-js/faker/discussions) or join our [Discord Chat Server](https://chat.fakerjs.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.
- [ ] I am willing to provide a PR.

### Describe the bug

https://github.com/faker-js/faker/blob/next/README.md?plain=1#L84

The above link directs to a README line which mentions a commented out bit of code for importing faker from a single locale.  However that bit of code does not exist!

### Environment Info

```shell
N/A
```

### Which module system do you use?

- [ ] CJS
- [ ] ESM

### Used Package Manager

npm

**Comment by matthewmayer** [comment]
We could probably remove this whole paragraph, it's already explained better at https://fakerjs.dev/guide/localization.html#individual-localized-packages:

```
The above code indicates a basic usage of Faker.
The point of interest is the import statements at the top.
The first import indicates how one can import the entirety of Faker, which includes every locale, while the commented-out import showcases how to import only a single locale.
In most situations, importing a single locale is preferable for performance because some testing frameworks reload imports for every test file, which causes startup latencies to add up quickly.
Thus, limiting the import to a single locale can speed up startup times.
```

**Comment by HenryHall** [comment]
It looks like it was [intentionally included](https://github.com/faker-js/faker/pull/1629/files#diff-ebc191ae76e5fef17b9b552067d614e7a84ad748c0d917f2486f9a61b5b8adfdR10) in this section of the readme to 'hint' at options for better performance.

**Comment by ST-DDT** [comment]
Sorry, for the late response.
The section is indeed outdated and should be removed, because since v8.0 faker instances only contain a single locale and not multiple as before.

Will be fixed in #2433

- #2433
